content/tracing/_index.md

```diff
@@ -2,75 +2,112 @@
+| File setting            | Environment variable | Description                                                                                                                                                      |
+|------------------------------|----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **main**                |                      |                                                                                                                                                                  |
+| `apm_enabled`           | `DD_APM_ENABLED`     | The Datadog Agent accepts trace metrics when the value is set to `true`. The default value is `true`.                                                            |
+| **trace.sampler**       |                      |                                                                                                                                                                  |
+| `extra_sample_rate`     | -                    | Use this setting to adjust the trace sample rate. The value should be a float between `0` (no sampling) and `1` (normal sampling rate). The default value is `1` |
+| `max_traces_per_second` | -                    | The maximum number of traces to sample per second. To disable the limit (*not recommended*), set to `0`. The default value is `10`.                              |
+| **trace.receiver**      |                      |                                                                                                                                                                  |
+| `receiver_port`         | `DD_RECEIVER_PORT`   | The port that the Datadog Agent's trace receiver should listen on. The default value is `8126`.                                                                  |
+| `connection_limit`      | -                    | The number of unique client connections to allow during one 30 second lease period. The default value is `2000`.                                                 |
+| **trace.ignore**        |                      |                                                                                                                                                                  |
+| `resource`              | `DD_IGNORE_RESOURCE` | A blacklist of regular expressions to filter out Traces by their Resource name.                                                                                  |
 {{% /table %}}
 
 For more information about the Datadog Agent, see the [dedicated doc page](/agent/) or refer to the [`datadog.conf.example` file](https://github.com/DataDog/dd-agent/blob/master/datadog.conf.example).
 
+## Defining namespace for trace metrics 
+
+The namespace is basically `trace.<name>.<metrics>{<tags>}` where
```

Comment:
Should we remove the word 'basically'? Sounds a little bit more formal.

```
The namespace is `trace.<name>.<metrics>{<tags>}` where
```

Reply:
👍